src/_component/connect/connect.scss

```diff
@@ -0,0 +1,147 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap");
+
+.space[height="30"] {
```

Comment:
You are repeating yourself in here the class space you got it in welcome.scss as well. Consider refactoring.

Reply:
refactor done